env.template

```diff
@@ -1,10 +1,20 @@
 # host download path
 HOST_DOWNLOAD_PATH=/tmp/
 
-# default user and password
+# nefarious user and password
 NEFARIOUS_USER=admin
 NEFARIOUS_PASS=admin
 
 # number of workers for searching and downloading.
 # leave as "0" to use the maximum number of CPUs on the system
 NUM_CELERY_WORKERS=0
+
+#########################################################
+# VPN Settings - leave blank if not using a vpn
+# https://haugene.github.io/docker-transmission-openvpn/
+#########################################################
+OPENVPN_PROVIDER=
+OPENVPN_USERNAME=
+OPENVPN_PASSWORD=
+OPENVPN_CONFIG=
```

Comment:
This env variable is not exposed to the vpn docker container per default. I think it definitely should be!

Reply:
whoops! You're right.  I swear I tested it so I'm at a loss as to how it was working.  I'll add it.

Reply:
I created a PR: #116